apsuite/loco/main.py

```diff
@@ -200,41 +203,61 @@ def _handle_quad_fit_k(self, fname_jloco_k_quad):
                     jloco_k_quad_dict, 'quadrupole')
 
     def _convert_dict2array(self, jlocodict, magtype, is_normal=True):
-        jloco = []
+
+        if 'knobs_order' in jlocodict:
+            knobs_order = jlocodict['knobs_order']
```

Comment:
I'm kind of sad you didn't choose to go with the `OrderedDict` here. I think it would be more pythonic and of ease transition for the ordinary `dict` once we change python's version to `>=3.7`.
However, since this is a temporary fix, I don't think it will be a problem.

Reply:
You are right, I'm sorry for disappointing you, but I intend to implement the more adequate solutions ASAP.

Reply:
ok!